addons/hr_holidays_attendance/models/hr_leave.py

```diff
@@ -93,6 +93,11 @@ def _remove_resource_leave(self):
         self._update_leaves_overtime()
         return res
 
+    def _create_resource_leave(self):
+        res = super()._create_resource_leave()
+        self._update_leaves_overtime()
+        return res
```

Comment:
I feel like this should be done from the `resource.calendar.leaves` model on create/write/unlink, as public leaves are directly created as that model and should also update overtimes if I'm not mistaken